packages/core/helper-plugin/lib/src/hooks/useTracking/index.js

```diff
@@ -0,0 +1,25 @@
+import { useContext, useRef } from 'react';
+import axios from 'axios';
+import TrackingContext from '../../contexts/TrackingContext';
+
+const useTracking = () => {
+  const trackRef = useRef();
+  const uuid = useContext(TrackingContext);
+
+  trackRef.current = (event, properties) => {
+    if (uuid) {
+      try {
+        axios.post('https://analytics.strapi.io/track', {
+          event,
+          properties: { ...properties, projectType: process.env.STRAPI_ADMIN_PROJECT_TYPE },
+        });
+      } catch (err) {
+        // Silent
+      }
+    }
+  };
+
+  return { trackUsage: trackRef.current };
```

Comment:
Can we directly do `return trackRef.current` as we do not return anything else? WDYT? So that we don't need to destructure the object every time we need to use the tracker.

Reply:
Initially I thought the exactly as you did but then making the hook returning an object makes sure that we use a consistent naming in the codebase so it will prevent us from having `track, emitEvent` as naming in the admin. It is easier to identify all the events that are sent.

WDYT?